### ISSUE TITLE
Confusing OpenEVSE Wifi version

### ISSUE THREAD

**Issue by dml33** [OpenEVSE/openevse_esp32_firmware#822]
Hi.

I have just updated with the huzzah32 binary from 5.0.2 and I have just noticed that the OpenEVSE Wifi version is quite confusing. Instead of 5.1.0, the github hash `master_d92677f6_modified` is shown:

![image](https://github.com/OpenEVSE/openevse_esp32_firmware/assets/25033130/3ece6fd1-85ae-485c-aa7a-5f4b18524079)

So, even having the 5.1.0, it shows as if it not already updated...

![image](https://github.com/OpenEVSE/openevse_esp32_firmware/assets/25033130/df8602f8-9a39-4c80-9d6a-66f28f139f21)

In case of the wifi_v1, the name shown is ` v5.1.0_modified`

I think it would be nice to have a clear name, like it was in 5.0.2: `v5.0.2`

**Comment by jeremypoulter** [comment]
Oh right, I know why. The release GUI module was not updated properly before the release, and then the missing binary was uploaded from the wrong source. I will do a new release

**Comment by SergioRius** [comment]
> Oh right, I know why. The release GUI module was not updated properly before the release, and then the missing binary was uploaded from the wrong source. I will do a new release

I hope that also fixes the lack of ethernet for the Olimex. I'm waiting on 5.0.1 (8.2.2EU) but it has important issues, like having to open the box to manually reset the openevse after a blackout or the shaper not working. :"(

**Comment by jeremypoulter** [comment]
this has been resolved in later releases
